Fix bug in `rows_patch.tbl_lazy()` throwing an error when multiple columns need to be patched (#1444)
tidyverse/dbplyr: NEWS.md

# dbplyr (development version)

> * `rows_patch()` no longer throws an error when more than one column needs to be
>   patched (@gorcha, #1443).

Reviewer: This should mention that this fix is for `in_place = FALSE` and I think it is a bit nicer to state it positively ("now works" instead of "no longer throws an error")

```suggestion
* `rows_patch(in_place = FALSE)` now works when more than one column should be
  patched (@gorcha, #1443).
```
